src/core/core.scale.js

```diff
@@ -495,10 +495,10 @@ module.exports = function(Chart) {
 			var min = me.min;
 			var max = me.max;
 
-			return me.beginAtZero ? 0:
-				min < 0 && max < 0? max :
-				min > 0 && max > 0? min :
-				0;
+			return me.beginAtZero ? 0 :
```

Comment:
Is this a new rule? I don't think it makes the code more readable, I would disable it